modules/stanford_profile_drush/src/Commands/StanfordProfileCommands.php

```diff
@@ -0,0 +1,270 @@
+   *   React paragraphs field name.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  protected function generateStressTestContent($entity_type_id, $bundle, $field) {
+    $label_key = $this->entityTypeManager->getDefinition($entity_type_id)
+      ->getKey('label');
+    $bundle_key = $this->entityTypeManager->getDefinition($entity_type_id)
+      ->getKey('bundle');
+
+    /** @var \Drupal\field\FieldConfigInterface $field_config */
+    $field_config = $this->entityTypeManager->getStorage('field_config')
+      ->load("$entity_type_id.$bundle.$field");
+
+    $this->entityTypeManager->getStorage($entity_type_id)->create([
+      $label_key => 'Stress Test ' . date('F j Y'),
+      $bundle_key => $bundle,
+      $field => $this->getParagraphFieldValues($field_config),
+    ])->save();
+  }
+
+  /**
+   * Get the field value for the react paragraphs field with sample content.
+   *
+   * @param \Drupal\field\FieldConfigInterface $field
+   *   React paragraphs field config object.
+   *
+   * @return array
+   *   Field value list array.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  protected function getParagraphFieldValues(FieldConfigInterface $field) {
+    $paragraph_bundles = $this->bundleInfo->getBundleInfo('paragraph');
+    $handler_settings = $field->getSetting('handler_settings');
+
+    $paragraphs = [];
+    foreach (array_keys($paragraph_bundles) as $bundle) {
+      // Find out if the field is configured to allow the current bundle. The
+      // field allows for "Exclude selected" which is the `negate` value. If a
+      // paragraph has been added but the field settings haven't been resaved,
+      // it may or may not be allowed in the field, so we check that too.
+      if (isset($handler_settings['target_bundles_drag_drop'][$bundle])) {
+        if ((bool) $handler_settings['negate'] === (bool) $handler_settings['target_bundles_drag_drop'][$bundle]['enabled']) {
+          continue;
+        }
+      }
+      elseif (!$handler_settings['negate']) {
+        continue;
+      }
+
+      // Create a paragraph entity that we can clone so we can see the effects
+      // at the different widths on the same display.
+      $original_paragraph = $this->createParagraph($bundle);
+      $paragraphs[] = [
+        'target_id' => $original_paragraph->id(),
+        'target_revision_id' => $original_paragraph->getRevisionId(),
+        'settings' => [
+          'row' => count($paragraphs),
+          'index' => 0,
+          'width' => 12,
+        ],
+      ];
+
+      // Take the original paragraph, clone it and make it smaller.
+      foreach ([6, 4, 3] as $width) {
```

Comment:
This is smrt.

Reply:
I was thinking about this. to allow for us to edit the page i'm going to clone the same component and put multiple of the same thing in the row.

Reply:
I made a change to clone the same paragraph and put it same row multiple times. I also added an admin label. It'll be funny because the "Spacer" paragraph will also generate rows but it will be empty.

Reply:
This is fun: https://share.getcloudapp.com/geuWvBqR